core/src/test/java/nl/nn/adapterframework/jdbc/dbms/TestBlobs.java

```diff
@@ -174,42 +207,58 @@ public void testWriteAndReadClobUsingDbmsSupport(int numOfBlocks, int blockSize)
 
 	}
 
-	@Test
-	public void testWriteAndReadBlobUsingDbmsSupport15MB() throws Exception {
-		testWriteAndReadBlobUsingDbmsSupport(10000,1500);
+	@DatabaseTest
+	public void testWriteAndReadBlobUsingDbmsSupport15MB(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		testWriteAndReadBlobUsingDbmsSupport(10000, 1500, databaseTestEnvironment);
 	}
 
-	@Test
-	public void testWriteAndReadBlobUsingDbmsSupport20MB() throws Exception {
+	@DatabaseTest
+	public void testWriteAndReadBlobUsingDbmsSupport20MB(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
 		assumeTrue(testBigBlobs);
-		assumeFalse("MariaDb cannot handle statement packets> 16M", productKey.equals("MariaDB"));
-		testWriteAndReadBlobUsingDbmsSupport(10000,2000);
+		assertNotEquals("MariaDB", dataSourceName, "MariaDb cannot handle statement packets> 16M");
+		testWriteAndReadBlobUsingDbmsSupport(10000, 2000, databaseTestEnvironment);
 	}
 
-	@Test
-	public void testWriteAndReadBlobUsingDbmsSupport100MB() throws Exception {
+	@DatabaseTest
+	public void testWriteAndReadBlobUsingDbmsSupport100MB(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
 		assumeTrue(testBigBlobs);
-		assumeFalse("MariaDb cannot handle statement packets> 16M, PostgreSQL uses ByteArray", productKey.equals("MariaDB") || productKey.equals("PostgreSQL"));
-		testWriteAndReadBlobUsingDbmsSupport(10000,10000);
+		assertFalse(dataSourceName.equals("MariaDB") || dataSourceName.equals("PostgreSQL"), "MariaDb cannot handle statement packets> 16M, PostgreSQL uses ByteArray");
+		testWriteAndReadBlobUsingDbmsSupport(10000, 10000, databaseTestEnvironment);
 	}
 
-	@Test
-	public void testWriteAndReadClobUsingDbmsSupport15MB() throws Exception {
-		testWriteAndReadClobUsingDbmsSupport(10000,1500);
+	@DatabaseTest
+	public void testWriteAndReadClobUsingDbmsSupport15MB(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
+		testWriteAndReadClobUsingDbmsSupport(10000, 1500, databaseTestEnvironment);
 	}
 
-	@Test
-	public void testWriteAndReadClobUsingDbmsSupport20MB() throws Exception {
+	@DatabaseTest
+	public void testWriteAndReadClobUsingDbmsSupport20MB(DatabaseTestEnvironment databaseTestEnvironment) throws Exception {
 		assumeTrue(testBigBlobs);
-		assumeFalse("MariaDb cannot handle statement packets> 16M", productKey.equals("MariaDB"));
-		testWriteAndReadClobUsingDbmsSupport(10000,2000);
+		assertNotEquals("MariaDB", dataSourceName, "MariaDb cannot handle statement packets> 16M");
```

Comment:
Oh en hier is hij helemaal anders. Assume kijkt of een voorwaarde klopt, anders gaat hij niet verder met de test.